src/main/java/com/example/todowithcouchbase/common/exception/GlobalExceptionHandler.java

```diff
@@ -67,10 +69,12 @@ protected ResponseEntity<Object> handlePathVariableErrors(final ConstraintViolat
                 );
 
         CustomError customError = CustomError.builder()
+                .time(LocalDateTime.now())
                 .httpStatus(HttpStatus.BAD_REQUEST)
                 .header(CustomError.Header.VALIDATION_ERROR.getName())
                 .message("Constraint violation")
                 .subErrors(subErrors)
+                .isSuccess(false)
```

Comment:
Bu builder'da tekrarlayan şekilde time ve isSuccess alanları aynı değerler ile setlenmiş. Kod tekrarının önüne geçmek için bu iki alana `@Builder.Default` ile default değerler atayabiliriz.

Reply:
Gerekli düzeltmeler yapıldı.

Reply:
Kontrolleri sağlayıp final değişkenler için proje bazında düzenlemeler yapılacak tamamdır